cpp/src/table/device_table.cuh

```diff
@@ -36,8 +36,9 @@
  * size
  *
```

Comment:
It appears changes to device_table.cuh in this PR are purely cosmetic -- changing the location of `{` and spacing to your preference. That makes it hard to review (hard to see the forest for the trees). Suggest leaving cosmetic things like this alone if they don't affect indentation (e.g. mixing tabs and spaces in the same file).

Reply:
Sorry... I was trying to clean up my formatting by using the IDE's auto-formatting, but I can see how that makes it harder to review the changes. I will revert that